rasa/nlu/featurizers/dense_featurizer/convert_featurizer.py

```diff
@@ -45,6 +45,12 @@ def __init__(self, component_config: Dict[Text, Any] = None) -> None:
 
         self._load_model()
 
+        logger.warning(
```

Comment:
this as a warning is confusing if people don't use sequences, can we move it to `_combine_with_existing_dense_features`? we need this kind of warning there anyhow